compiler/src/scala/quoted/runtime/impl/QuotesImpl.scala

```diff
@@ -1826,7 +1826,7 @@ class QuotesImpl private (using val ctx: Context) extends Quotes, QuoteUnpickler
         def termSymbol: Symbol = self.termSymbol
         def isSingleton: Boolean = self.isSingleton
         def memberType(member: Symbol): TypeRepr =
-          member.info.asSeenFrom(self, member.owner)
+          member.info.asSeenFrom(self, member.owner, approximateUnstablePrefixes = false)
```

Comment:
No, I don't think we should do it this way.  Perhaps we should do something like `member.info.asSeenFrom(self.<thisType>, member.owner)` .